.coveragerc

```diff
@@ -10,6 +10,10 @@ include =
   *\Lib\site-packages\pytest.py
 parallel = 1
 branch = 1
+# From https://pypi.org/project/coverage_enable_subprocess/
+# $TOP env var is emitted during the tests run
```

Comment:
Out of curiosity, who sets `TOP`? I don't see it in `coverage_enable_subprocess`' sources, is it defined by the system?

Reply:
let me look into this - this one was left from my initial iteration, it could be that the absolute path to coverage file in CI is the culprit.